src/prefect/cli/agent.py

```diff
@@ -0,0 +1,65 @@
+import click
+
+from prefect import config
+from prefect import agent as prefect_agent
+from prefect.agent.kubernetes import KubernetesAgent
+from prefect.utilities.configuration import set_temporary_config
+
+_agents = {
+    "local": prefect_agent.local.LocalAgent,
+    "kubernetes": KubernetesAgent,
+    "nomad": prefect_agent.nomad.NomadAgent,
+}
+
+
+@click.group(hidden=True)
+def agent():
+    """
+    Manage Prefect agents.
+
+    \b
+    Usage:
+        $ prefect agent [COMMAND]
+
+    \b
+    Arguments:
+        start       Start a Prefect agent
+
+    \b
+    Examples:
+        $ prefect agent start
+
+    \b
+        $ prefect agent start kubernetes --token MY_TOKEN
+    """
+    pass
+
+
+@agent.command(hidden=True)
+@click.argument("name", default="local")
+@click.option(
+    "--token", "-t", required=False, help="A Prefect Cloud API token.", hidden=True
```

Comment:
Looks like this flag doesn't do anything?

Reply:
Yeah it's used to overwrite the value in config for that particular agent run if it is specified